src/script/components/MessagesList/Message/MessageWrapper.tsx

```diff
@@ -215,6 +215,7 @@ export const MessageWrapper: React.FC<MessageParams & {hasMarker: boolean; isMes
         onRetry={onRetry}
         isMessageFocused={isMessageFocused}
         isMsgElementsFocusable={isMsgElementsFocusable}
+        messageRepository={messageRepository}
```

Comment:
Not all message have reaction, only content message can have reaction. so it made more sense to me to have it inside contentmessage instead of message wrapper.

Reply:
The problem is, this injects too much power in the `ContentMessage`. It should never have full access to the `MessageRepository` this is not going to scale. 
Only a handful of components should have access to it. In this case it's the `MessageWrapper`